src/AccountIcon.tsx

```diff
@@ -1,22 +1,12 @@
-import * as React from 'react';
```

Comment:
We have to update all the icon component, because React import is not needed on v18